src/Platform/ACSFPlatform.php

```diff
@@ -195,21 +198,41 @@ public function execute(Command $command, InputInterface $input, OutputInterface
     [$application] = explode('.', $url);
     $output->writeln(sprintf("Attempting to execute requested command in environment: %s", $environment->uuid));
     $commands = [];
-    if ($input->hasOption('uri') && $uri = $input->getOption('uri')) {
+
+    $sites = $this->getPlatformSites();
+    if (!$sites) {
+      $output->writeln('<warning>No sites available. Exiting...</warning>');
+      return 1;
+    }
+
+    $group_name = $input->getOption('group');
+    $uri = $input->getOption('uri');
+
+    if ($group_name && $uri) {
+      $helper = $command->getHelper('question');
+      $question = new ConfirmationQuestion('You have provided both the options, group as well as uri. We will ignore the uri option. Do you want to proceed (y/n)?', TRUE);
```

Comment:
This is problematic when a command runs internally through PlatformCommand Executioner in which case user isn't shown this question and execution kind of halts without informing anything to the user.

Let's ignore group and only use uri in this case without any choice question.

Reply:
Done.